### PR TITLE
[WIP] Update ruby/rails

### PR DESCRIPTION
**Pull request by ArthurZaharov**

### CODE REVIEW

File: app/models/error.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require "securerandom"
```
Comment (ArthurZaharov) [inline app/models/error.rb]: Do we need it?

File: bin/yarn [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```
Comment (ArthurZaharov) [inline bin/yarn]: Do we need this file?

File: config/initializers/filter_parameter_logging.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += %i(password auth_token)
+Rails.application.config.filter_parameters += [:password]
```
Comment (ArthurZaharov) [inline config/initializers/filter_parameter_logging.rb]: According to knock gem (https://github.com/nsarno/knock/blob/master/lib/knock/authenticable.rb#L11) `token` can be passed through parameter. Should we filter it to here?

File: config/initializers/generators.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-app_config.app_generators do |g|
+Rails.application.config.app_generators do |g|
   g.fixture_replacement :factory_girl, dir: "spec/factories"
```
Comment (ArthurZaharov) [inline config/initializers/generators.rb]: Should be factroy_bot as I think.

File: config/initializers/wrap_parameters.rb [hunk 5]
```diff
@@ -5,8 +5,10 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
```
Comment (ArthurZaharov) [inline config/initializers/wrap_parameters.rb]: We do not need it for jsonapi requests as I think.

File: .env.example [hunk 6]
```diff
@@ -1,6 +1,37 @@
...
+
+# Enable basic auth to close the app from unauthorized viewers
+# AUTH_BASIC_REALM=your_application_name
+# AUTH_BASIC_PASS=your_password_here
+
+# Set single hostname
+# CANONICAL_HOST=paste_single_hostname_here
+
+# Comma separated list of IPs to access admin staff like RackMiniProfiler, Sidekiq Web, Flipper UI
+IP_WHITELIST=127.0.0.1
```
[the remark this answers is not in the document]
Comment (timurvafin) [reply .env.example]: Do you think we need to add RackMiniProfiler now?

**Comment by timurvafin** [comment]
Closed in favor of https://github.com/fs/rails-base-api/pull/230

**Pull request closed without merging**